modules/kvantum/kvconfig.mustache

```diff
@@ -0,0 +1,466 @@
+[%General]
+author=Bluskript based on Catppuccin Frappe Mauve theme
```

Comment:
```suggestion
author={{scheme-author}}
```

Reply:
Actually not sure whether this is better as the scheme author, or how it is now. I'll leave it up to you whether you want to change this :)